pkg/chain/gen/celo/contract/BondedECDSAKeep.go

```diff
@@ -238,7 +238,7 @@ func (becdsak *BondedECDSAKeep) DistributeERC20Reward(
 ) (*types.Transaction, error) {
 	becdsakLogger.Debug(
```

Comment:
Was this written before we started using `Debugf`? Want to change it or leave it in?

Reply:
oh wait this fine is generated @_@